rust/main/chains/hyperlane-sealevel/src/rpc/client.rs

```diff
@@ -513,7 +513,7 @@ impl SealevelRpcClient {
             Transaction::new_signed_with_payer(
                 &instructions,
                 Some(&payer.pubkey()),
-                &[payer],
+                &[payer.keypair()],
```

Comment:
wonder if you need to do this .keypair() or not

Reply:
I tried without it, but it was giving me errors. So not exactly sure what was happening :eyes: